tmt/steps/execute/internal.py

```diff
@@ -118,7 +123,7 @@ def _show_progress(self, progress, test_name, finish=False):
             self._previous_progress_message = ""
         sys.stdout.flush()
 
-    def _test_environment(self, test, extra_environment):
+    def _test_environment(self, test: Test, extra_environment: Dict[str, Any]) -> Dict[str, Any]:
```

Comment:
I'd say we could use `EnvironmentType` from `tmt.utils` here, instead of `Dict[...]`,